plugins/polio/js/src/domains/LQAS-IM/shared/Map/LqasImPopUp.tsx

```diff
@@ -29,7 +43,9 @@ export const LqasImPopup = ({ shape, data, round, campaign }) => {
         campaign,
     });
     return dataForShape ? (
-        <Popup className={classes.popup} ref={ref} pane="popupPane">
+        // removing the pane prop causes zIndex bug
+        // @ts-ignore
+        <Popup ref={ref} pane="popupPane">
```

Comment:
I don't get it , you removed the className prop, not the pane one

Reply:
Yeah, but TS complains about it, so the comment aims to explain why it needs to stay, also TS says the prop doesn't exist